index.html

```diff
@@ -0,0 +1,66 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Project3</title>
+  </head>
+  <body>
+    <header>
+      <h1>
+        my conference header
+      </h1>
+    </header>
+    <nav>
+      <ul>
+        <li><a href="#About">About</a></li>
+        <li><a href="#Schedule">Schedule</a></li>
+        <li><a href="#Speakers">Speakers</a></li>
+        <li><a href="#Register">Register</a></li>
+      </ul>
+    </nav>
+    <main>
+      <p id="About">
+        <!--Site content. Conference Descriptions, images, etc.-->
+        <!-- About -->
+        This is a conference for every designers who uses types and letterings.
+        This year's conference will be hosted online on February 28, 2021.
+      </p>
+      <p id="Schedule">
+        Conference will be on Feb.28, 2021 9:00 A.M.- 7:00 P.M. 9:00 A.M.:
+        Introduction 10:00 A.M.- 7:00 P.M.: Typographic/Lettering Design Fair
+        10:30 A.M.: Hand Lettering workshop 1:00 P.M.: Typeface as a design tool
+        2:30 P.M.: Visual Typography
+      </p>
+      <p id="Speakers">
+        Carol Twombly- During her eleven+ years with Adobe, Carol has designed a
+        number of very popular text and display typefaces. Designs like Trajan,
+        Charlemagne, Lithos, and Adobe Caslon are inspired by classic
+        letterforms of the past - from early Greek inscriptions, circa 400 B.C.,
+        to William Caslon’s typefaces of the 1700s. Designs like Viva and Nueva
+        explore new territory while maintaining traditional roots. In 1994, she
+        received the Charles Peignot award from the Association Typographique
+        Internationale for outstanding contributions to type design. She was the
+        first woman and only the second American to receive this prestigious
+        honor.
+        Mary Kate McDevitt- A letterer and illustrator, McDevitt lives
+        and works in Brooklyn, New York. Some of her prominent clients include
+        AT&T, Penguin Books, Nike, United States Postal Service, Sesame Street,
+        and Target. She also worked with Chronicle Books, Nintendo, and Wall
+        Street Journal, as well as Smart Ones, Fast Company and O Magazine.
+        McDevitt had a degree in Graphic Design and Illustration from Tyler
+        School of Art in Philadelphia.
+        Marian Bantjes- Having worked as a book
+        typesetter and founding the Digitopolis graphic design studio, Canadian
+        designer Marian Bantjes currently works on her own projects in design,
+        art, and typography. Her ornamental, often dreamlike style has been
+        featured in over 100 books, including her own beautiful I Wonder in 2010
+        (with a foreword by Stefan Sagmeister of the aforementioned Sagmeister &
+        Walsh). Bantjes continues to bring new and unusual letterforms into the
+        design vocabulary.
+      </p>
+      
+    </main>
+  </body>
+</html>
```

Comment:
I know your HTML is still something a work in progress, so I'm just listing the requirements here to help you along:

The design must include:
-    Event name ✅ I expect this will end up in your `h1` on line 12
-    In-page navigation ✅ 
-    Event schedule ✅ 
-    Details for at least three talks (including talk name and description) ◻️
-    Three speaker bios (including a photo) ✅ Photo still to be added
-    Call to action and link to buy tickets ◻️

Your code must include the following HTML elements:
- header ✅ 
- main ✅ 
- footer
- h1 ✅ 
- img
- a ✅